azerothcore/azerothcore-wotlk
# fix(Core/Spell): Eye of Gruul healing discount spell

Fix triggered spell from casting healing spells wearing Eye of Gruul (http://wotlk.cavernoftime.com/item=28823)


##### CHANGES PROPOSED:
-  Improve PR https://github.com/azerothcore/azerothcore-wotlk/pull/1764
-  Benefit all healing classes wearing Eye of Gruul

**TESTS PERFORMED**: Build without errors and tested in-game. (Win10 and Debian 9 servers).

##### HOW TO TEST THE CHANGES:

 - Create healing class lvl >=70
 - .add 28823
 - Cast healing spells until the item procs Healing discount (Could take a while xD)
 - The next healing spell cost should reduced by 450 while Healing discount buff lasts


##### Target branch(es):

Master
